src/main/java/ShortSet.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class ShortSet {
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Error Prone also found similar issues in RecommenderTest/src/main/java/LongSet.java:3: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class LongSet

RecommenderTest/src/main/java/LongSet.java:3: error: cannot find symbol
    Set<Short> s = new HashSet<>();
                       ^
  symbol:   class HashSet
  location: class LongSet

RecommenderTest/src/main/java/ShortSet.java:4: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class ShortSet

RecommenderTest/src/main/java/ShortSet.java:4: error: cannot find symbol
    Set<Short> s = new HashSet<>();
                       ^
  symbol:   class HashSet
  location: class ShortSet

Check out http://errorprone.info for more information.


[1] ```RecommenderTest/src/main/java/ShortList.java:3: error: cannot find symbol
    List<Short> s = new ArrayList<>();
    ^
  symbol:   class List
  location: class ShortList```